game_session.js

```diff
@@ -1,6 +1,9 @@
 const Scoreboard = require("./scoreboard.js");
 const BEGINNING_SEARCH_YEAR = 2008;
-const DEFAULT_OPTIONS = { beginningYear: BEGINNING_SEARCH_YEAR }
+const MALE = "male";
+const FEMALE = "female";
+const COED = "coed";
```

Comment:
Might be cleaner to save these under an object like `GENDER.MALE`, `GENDER.FEMALE`, etc